bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -171,7 +171,9 @@ <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get ev
 
 {% block newsletter_modal %}{% endblock %}
 
-{% include 'firefox/includes/schemaorg-app.html' %}
+{% if LANG == 'en-US' %}
```

Comment:
It looks like this is in the wrong template. You might want to try `/firefox/new/trailhead/download.html`?

Reply:
I believe if you rebase this branch against master, there should also now be a `{% block structured_data %}` block in the [base templates](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-pebbles.html#L128-L134) that you can use for positioning here in the template.